wheelhouse.txt

```diff
@@ -0,0 +1 @@
+pydantic <= 2.10.6
```

Comment:
As you have noted in your commit message, interfaces do not have influence on charm wheelhouse.

Reactive interfaces had a tendency to be very light weight and have no dependencies apart from the standard python library.

This is different with ops interfaces though, so we probably just have to accept this and document requirements for charm author to include.